### PR TITLE
Updated instructions of establishing HRMP channel post-XCM v3

### PR DESCRIPTION
**Pull request by chrisli30**
Updated screenshot and encoded call data for the example of `hrmp.hrmpInitOpenChannel()` request.

### CODE REVIEW

**Comment by chrisli30** [comment]
@imstar15 could you take a quick look at this PR?

File: _docs/xcm/hrmp-channel.md [hunk 1]
```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
...
+Sovereign Account (Parachain): `0x7369626c42080000000000000000000000000000000000000000000000000000`
 
 ## Turing Network
-Encoded account: `0x7061726142080000000000000000000000000000000000000000000000000000`\
-[Turing open channel call](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/extrinsics/decode/0x29000101000214000400000000070088526a741300000000070088526a7400060003005ed0b2183c01dc070000140d010004000101007061726142080000000000000000000000000000000000000000000000000000) - do this call from sudo tab or via governance.
+The below details can be viewed on [Kusama Subscan](https://kusama.subscan.io/parachain/2114).
+
+Sovereign Account (Kusama): `F7fq1is7XWry4tWaYVZJ2uSwytCLFhsuEFarS7hit1chXtq`
+
+Sovereign Account (Turing Network): `67y2xkd8rZD7gapMM7EA8pug4rSvUCQjaoTm95mVEcTiR3w1`
```
Comment (imstar15) [inline _docs/xcm/hrmp-channel.md:149]: Does this line mean the Sovereign account on Turing?

It should be `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn`.

File: _docs/xcm/hrmp-channel.md [hunk 2]
```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
 ```
 
 ## Turing Staging
-Parachain sovereign account: 5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y
+The below details can be viewed on [Rococo Subscan](https://rococo.subscan.io/parachain/2114).
+
+Sovereign Account (Rococo V1): `5Ec4AhNzVAVwDR54oqgWQ5YSoPbxXathSrQqTmqkF5o7xe6y`
+
+Sovereign Account (Turing Staging): `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`
```
Comment (imstar15) [inline _docs/xcm/hrmp-channel.md:140]: Does this line mean the Sovereign account on Turing Staging?

It should be `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn`.

File: _docs/xcm/hrmp-channel.md [hunk 3]
```diff
@@ -133,9 +133,26 @@ tokenDecimals: 10
...
+
+Sovereign Account (Parachain): `0x7369626c42080000000000000000000000000000000000000000000000000000`
```
Comment (imstar15) [inline _docs/xcm/hrmp-channel.md:142]: Does this line mean the Sovereign account on Parachain?

It should be `0x7061726142080000000000000000000000000000000000000000000000000000`.

File: _docs/xcm/hrmp-channel.md [hunk 4]
```diff
@@ -60,32 +60,32 @@ The first step is to prepare an encoded call data for the payload. Here we are u
...
 
-If you have a **sudo** account on your chain, you could construct the below polkadotXcm.send() call right from it. However, since Turing Staging has removed sudo, we are using Governance to execute the call below.
+If you have a **sudo** account on your chain, you can use it to construct the 
+polkadotXcm.send() call. However, since Turing Staging has removed sudo, we are using Governance to execute the call. For quick access, here is a link to an [Encoded call data example for polkadotXcm.send()](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/extrinsics/decode/0x29000301000314000400000000070010a5d4e81300000000070010a5d4e800060002286bee419c383c00e7070000e803000000900100140d01000001000921)
 
 1. Open chain A’s [polkadot.js app](https://polkadot.js.org/apps).
 2. Navigate to Governance -> Democracy, and click on Submit preimage.
 3. Within submit preimage popup, select `polkadotXcm.send` with the below parameters, copy the **preimage hash**, and submit.
    - destination: V1 {XcmV1MultiLocation { parents: 1, interior: Here}}
```
Comment (imstar15) [inline _docs/xcm/hrmp-channel.md:76]: Please change it to V3

**Review by imstar15 — CHANGES_REQUESTED** [review]

File: _docs/xcm/hrmp-channel.md (hunk 3 above)
Comment (chrisli30) [reply _docs/xcm/hrmp-channel.md:142]: Could you explain how was this value calculated?

File: _docs/xcm/hrmp-channel.md (hunk 2 above)
Comment (chrisli30) [reply _docs/xcm/hrmp-channel.md:140]: `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn` shows in Turing Staging as parachain 2114. We need to understand the difference between `68gzVdFcwnNUcwiExdbioBwGcktc2EZP9KQ1zvdTQn9WVdmn` and `5Dt6dpkWPwLaH4BBCKJwjiWrFVAGyYk3tLUabvyn4v7KtESG`

File: _docs/xcm/hrmp-channel.md (hunk 3 above)
Comment (chrisli30) [reply _docs/xcm/hrmp-channel.md:142]: It should be the sibling account on another parachain such as Shiden, so `0x7369626c42080000000000000000000000000000000000000000000000000000` is correct.

**Review by imstar15 — APPROVED** [review]
Looks good to me!

**Pull request merged**